backend/src/prompts/templates/answer-user-question.j2

```diff
@@ -9,23 +9,24 @@ User's question is:
 Once you generate an answer:
 - **Check** if the answer completely and accurately addresses the user's question.
 - **Determine** if the answer is valid, based on the content provided.
+- **Provide** a reason for the validity or invalidity of the answer.
+- **Indicate** if a more general web search is required then indicate that.
 
 Reply only in JSON format with the following structure:
 
 ```json
 {
     "answer": "The answer to the user's question, based on the content provided",
-    "is_valid": true or false,
-    "validation_reason": "A sentence explaining whether the answer is valid or not, and why"
+    "perform_web_search": true or false,
+    "perform_web_search_reason": "A sentence explaining whether the answer is valid or not, and why"
 }
 
 
-
 ### **Explanation:**
```

Comment:
the explanation seems to be very repetitive of the json - I wonder if we could just have the json explinations fleshed out and not have this ?